Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'rails', '~> 7.0.8'
 gem 'sprockets-rails'
 
 # Use sqlite3 as the database for Active Record
```

Comment:
| Migrations Order|
|-|
| ![image](https://github.com/abdozayan12/DataApp/assets/93039359/593655a7-4c83-4e15-9cb3-b4d1bc3de5a2) |

**[OPTIONAL BUT HIGHLY RECOMMENDED}**

- [ ] My apologies I forgot to mention one important point and it is not a big or difficult change but a child's play and very important too 🤔 Your order of migrations should be according to the project requirements because it creates a problem 😨 Because it goes top to down so when the migrations run then first it creates a comments table and gives us an error because your comments table finds post_id and user_id which doesn't exist because user table doesn't exist 🤔 So it would be great if you make your migrations in a correct order ⬇️

### 🔴 Incorrect Order
```
1. Comments Table
2. Posts Table
3. Likes Table
4. Users Table
```

### 🟢 Correct Order
```
1. Users Table
2. Posts Table
3. Comments Table
4. Likes Table
```